src/Seq.Input.HealthCheck/HealthCheckInput.cs

```diff
@@ -39,20 +40,27 @@ public class HealthCheckInput : SeqApp, IPublishJson, IDisposable
             InputType = SettingInputType.LongText)]
         public string TargetUrl { get; set; } = null!;
 
+        [SeqAppSetting(
+            DisplayName = "Follow Redirects",
+            IsOptional = true,
+            HelpText = "If selected, the HTTP Request will follow redirects. Defaults to false.",
```

Comment:
Nit: `Request` should probably not be captialised here.

Instead of "Defaults to false" could we say "By default, health checks that trigger redirects will fail."?